backend/src/test/java/com/huellapositiva/proposal/service/ProposalServiceShould.java

```diff
@@ -0,0 +1,82 @@
+package com.huellapositiva.proposal.service;
```

Comment:
The problem you are facing with the coverage is due to this test class being in the wrong package.

In our pipeline we have divided the testing phases in three: unit, integration and acceptance

These phases are executed independently and are configured in the POM file to merge their corresponding code coverage reports into a unique place. The thing is that in order to distinguish what tests belong to which phases, we configured the different testing phases to be bound to different packages.

So, unit tests are placed under `src/test/java/com/huellapositiva/unit`, integration tests under `src/test/java/com/huellapositiva/integration` and acceptance tests under `src/test/java/com/huellapositiva/api`

We could have used a different approach like naming the test classes differently depending on the test type e.g. FooTest, FooIT, FooAT 

You can see the relevant configuration in the POM and pipeline files.

To fix your issue move this test class into `src/test/java/com/huellapositiva/integration`

Reply:
Thanks a lot!